src/arguments/api_test.py

```diff
@@ -37,26 +38,56 @@ def get_request(cls):
             request_url += endpoint
 
         print("Trying ...\u26A1")
+        return {
+            "request_url" : request_url,
+            "request_headers" : request_headers,
+        }
 
-        # Make GET request and store the response in response_data.json
-        try:
-            response = requests.get(request_url, headers=request_headers)
-            print(f"Reponse Status Code: {response.status_code}")
-            response_data = json.loads(response.content)
-            parsed_json = json.dumps(response_data, indent=4)
-            output_json = highlight(parsed_json, lexers.JsonLexer(),
-                                    formatters.TerminalFormatter())
-            print(output_json)
-
-            store_data = input('Store response data? (Y/N): ')
-            if(store_data == 'Y' or store_data == 'y'):
+    #saves the json response into a file
+    @classmethod
+    def save_response_data(cls,response_data):
+        store_data = input('Store response data? (Y/N): ')
+        if(store_data == 'Y' or store_data == 'y'):
```

Comment:
Can we add an else statement as well saying that if he/she enters some other alphabet or word

Reply:
shall we print ``` You have entered {store_data}, please enter 'Y' or 'y' to save the output``` on the terminal

Reply:
But what if I enter some other alphabet then it will show an exception. We need to handle that

Reply:
I didn't get what you were saying, we are handling this in ```else ``` part only which checks he entered some other letter

Reply:
Yes